src/lib/metricsMgr/MetricsManager.h

```diff
@@ -107,24 +107,15 @@
 *
 * FIXME PR: summary of things to improve
 *
-* 01. Check alternatives to "triple std::map" from the point of view of performance
```

Comment:
If at the end we declare "remove" for this item I think we should remove some "FIXME PR" code existing code related with this.

Please, review the "FIXME PR" marks in MetricssManager.cpp in case someone could be removed.